@ecomplus/storefront-template/template/js/netlify-cms/base-config/collections/settings/stamps.js

```diff
@@ -30,24 +30,17 @@ export default ({ baseDir, state }) => ({
           }
         },
         {
-          label: 'Lista de produtos',
-          required: false,
+          label: 'Lista de skus de produtos',
           name: 'skus',
-          widget: 'list',
-          fields: [
-            {
-              label: 'SKU do produto',
-              required: false,
-              name: 'sku',
-              widget: 'string',
-              options: state.routes
-                .filter(({ sku }) => typeof sku === 'string')
-                .map(({ sku }) => ({
-                  label: sku,
-                  value: sku
-                }))
-            }
-          ]
+          required: false,
```

Comment:
Por que opcional? Se não tiver ele coloca o selo em todos os produtos?

Reply:
Não, porque toda vez que lanço algo que não é required, ele fica travando a configuração, se bem que não tem outros itens na view, então, talvez não tenha problema, vou mudar.

Reply:
você acha que deve ter a opção? tipo, se não tiver nada, colocar em todos os produtos, é valido ne

Reply:
Não é "sempre"

Reply:
É só quando você adiciona uma propriedade em um objeto que já existe, aqui é uma propriedade de um array, o objeto não existe em nenhuma loja porque o array por padrão é vazio

Reply:
Ou você mostra o selo por padrão em todos os produtos se não tiver lista de SKUs (ou se for vazia), ou esse field é required sim

Reply:
Valeu, pronto!